+sig/+node/Signal.m

```diff
@@ -286,53 +286,65 @@
-      % values directly in the target's struct value all done in mexnet
-      % according to the transfer opcode
+      % Returns a signal `fs` whose value is a struct containing 
+      % "flattened" fields for the signal fields in the struct that 
+      % `this` has as its value. 
+      %
+      % Uses a signal (whose value is a struct with signal fields) as a 
+      % blueprint to wire up signals as inputs to this target. The values
+      % of these signal fields will be directly set as the target signal's
+      % struct field values. This is done in mexnet according to the 
+      % transfer opCode.
+      
       fs = applyTransferFun(this, 'sig.transfer.flattenStruct', [], '%s.flattenStruct()');
-%       state = StructRef;
-%       state.unappliedInputChanges = false;
-%       state.inputsToSubsref = containers.Map(...
-%         'KeyType', 'uint64', 'ValueType', 'any');
-%       fs = applyTransferFun(this, 'sig.transfer.flattenStruct', state,...
-%         '%s.flattenStruct()');
+
     end
 
     function fs = flatten(this)
-      % all done in mexnet according to the transfer opcode
+
       state = StructRef;
       state.unappliedInputChanges = false;
       fs = applyTransferFun(this, 'sig.transfer.flatten', state,...
         '%s.flatten()');
     end
     
     function tr = applyTransferFun(varargin)
-      % New signal derived by applying a transfer function to input nodes
+      % New signal derived by applying a transfer function to input node(s)
+      % 
+      % This function creates a node from the nodes of input values (these 
+      % input values can be signals or non-signals), then creates a new 
+      % signal from the new node.
       %
-      % [tr] = s1.applyTransfer([s2], ..., funName, funArg, formatSpec)
-      % returns a new signal tr whose values result from applying a
-      % transfer function called funName to a variable number of input
-      % signals or constant values. Transfer functions work at a lower
-      % level than transformations like like map or mapn, instead operating
-      % directly with the underlying input nodes and output node,
-      % potentially using both their current *and* new values.
+      % Transfer functions work at a lower	level than transformations like 
+      % `map` or `mapn`, instead operating directly with the underlying 
+      % input nodes and output node,potentially using both their current 
+      % *and* new values.
       %
-      % This function creates a node with input nodes from each of the
-      % signals s1, s2,... (or for each non-signal values, creates a
-      % 'constant' node with that value). It then returns a sig.node.Signal
-      % tr containing the new node. It also sets the node's FormatSpec
-      % property as formatSpec.
+      % [tr] = s1.applyTransferFun([s2], ..., funName, funArg, formatSpec)
+      % 
+      % Inputs:
+      %   `varargin`: contains one (or more) input values/signals, `sigs`, 
```

Comment:
You don't really have to specify that 'varargin' is an input, that's sort of an irrelevant thing related to MATLAB's implementation of variable input arguments.  Users want to know what variables to pass into the function.  For readability you can have the inputs something like this:
```
Inputs:
      %   `sigs` : One or more Signal objects
      %   `funName` (char) : Name of the transfer function to use (optional; default: `sig.transfer.nop`)
```

Reply:
Another thing that might be worth checking is that these doc strings are rendered well in readthedocs